.github/workflows/pr-build-merge.yml

```diff
@@ -27,16 +27,33 @@ jobs:
       run: docker compose build app_test
 
     - name: "Create environment file"
-      run: touch .env
+      run: env | grep -E '^MPT_' > .env
+      env:
+        RP_ENDPOINT: ${{ secrets.RP_ENDPOINT }}
+        RP_API_KEY: ${{ secrets.RP_API_KEY }}
+        MPT_API_BASE_URL: ${{ secrets.MPT_API_BASE_URL }}
+        MPT_API_TOKEN: ${{ secrets.MPT_API_TOKEN }}
+
+    - name: "Run validation & test - no report"
+      run: docker compose run --service-ports app_test bash -c "ruff format --check . && ruff check . && flake8 . && mypy . && uv lock --check && pytest tests/unit"
+      env:
+        RP_LAUNCH: github-unit-test
+        RP_ENDPOINT: ${{ secrets.RP_ENDPOINT }}
+        RP_API_KEY: ${{ secrets.RP_API_KEY }}
 
     - name: "Run validation & test"
-      run: docker compose run --service-ports app_test
+      run: docker compose run --service-ports app_test bash -c "ruff format --check . && ruff check . && flake8 . && mypy . && uv lock --check && pytest --reportportal --rp-launch=$RP_LAUNCH --rp-api-key=$RP_API_KEY --rp-endpoint=$RP_ENDPOINT tests/unit"
```

Comment:
I don't think that it is required to report unit test also :-) we have sonar for that